doc/howto/write_a_view/index.md

```diff
@@ -0,0 +1,429 @@
+underlying range, we don't need to change the relation between iterator and sentinel, i.e. "begin == end" on
+our view iff "begin == end" on the underlying range.
+This indicates that we can use the sentinel of the underlying range as-is.
+
+\assignment{Exercise 3: Your first iterator}
+
+Have a peak at the \ref tutorial_concept tutorial again and study the `std::ForwardIterator` concept thoroughly.
+You will now have to implement your own forward iterator.
+
+\snippet doc/howto/write_a_view/solution_it.cpp start
+
+```cpp
+    // YOUR IMPLEMENTATION GOES HERE (OPERATORS, MEMBER TYPES...)
+```
+\snippet doc/howto/write_a_view/solution_it.cpp end
+
+It should wrap the iterator of the underlying range or inherit from it (shown above); in the end, you should be able
+to iterate over the underlying range and print elements like you would with the original iterator, i.e. your iterator
+shall behave exactly as the original in this regard (no transformation, yet).
+
+Some things to keep in mind for the implementation:
+  * When defining a type template (like above), member types are not inherited implicitly (member functions are).
+  * Another reason "just inheriting" is not sufficient, is that some inherited functions return objects or references
+    to the base type, not your type (which is required by the `std::ForwardIterator` concept).
+  * If you choose to wrap the underlying iterator instead of inheriting, you will need to define and "forward"
+    a few more member functions, but it's good practice and will help you understand the concept.
+  * The `static_assert` will just return `true` or `false`, to get more detailed information on why your type does
+    not (yet) model the concept, implement a constrained function template and pass an object of your type to that
+    template.
+  * Your iterator needs to be comparable to the sentinel of the underlying range, you can access that type via
+    `std::ranges::sentinel_t<urng_t>`.
+
+\endassignment
+
+\solution
+\snippet doc/howto/write_a_view/solution_iterator.cpp start
+\snippet doc/howto/write_a_view/solution_iterator.cpp solution1a
+```cpp
+    using reference             = typename std::iterator_traits<base_t>::reference;
+
+```
+\snippet doc/howto/write_a_view/solution_iterator.cpp solution1b
+\snippet doc/howto/write_a_view/solution_iterator.cpp end
+
+The program prints "G A T T A C A ".
+\endsolution
+
+\assignment{Exercise 4: A transforming iterator}
+
+In the previous assigment you have created a working, but pointless iterator.
+It does not do anything differently from the original.
+
+Your task now is to implement the "complementing" behaviour, i.e.
+  * your iterator should only work on ranges of `seqan3::NucleotideAlphabet`
+  * when accessing an element through the iterator it should not return the element from the underlying range, but
+    instead it's complement
+
+Think about which operator is responsible for returning the element and have special care with the return type
```

Comment:
```suggestion
Think about which operator is responsible for returning the element and take special care of the return type
```

Reply:
I meant "give a care" (*gib acht*) not "take care" (*kümmern*). But i will change it to something else altogether.

Reply:
Ok.